### PR TITLE
Fixup: Handle cgroup path for different distros

### PR DESCRIPTION
**Pull request by sathnaga**
Looks like libvirt cgroup path is different for
ubuntu distro, so handle multiple paths to resolve
libvirt cgroup folders.

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/cpu/guestpin.py [hunk 1]
```diff
@@ -109,24 +109,28 @@ def set_condition(vm_name, condn, reset=False, guestbt=None):
...
-                    machine_cpuset_cpus = os.path.join(machine_cpuset_path,
-                                                       "cpuset.cpus")
+                machine_cpuset_paths = []
+                if os.path.isdir(os.path.join(root_cpuset_path,
+                                 "machine.slice")):
+                    machine_cpuset_paths.append(os.path.join(root_cpuset_path,
+                                                             "machine.slice"))
+                if os.path.isdir(os.path.join(root_cpuset_path, "machine")):
+                    machine_cpuset_paths.append(os.path.join(root_cpuset_path,
+                                                             "machine"))
```
Comment (balamuruhans) [inline libvirt/tests/src/cpu/guestpin.py]: instead of handling multiple path can we handle root_cpu_path w.r.t to distro

Comment (sathnaga) [reply libvirt/tests/src/cpu/guestpin.py]: @balamuruhans I am not sure if it changes w.r.t distro solid, may be in future some configs can set it back to machine.slice, do not want to take a chance, no harm to set to all available paths in that level and we set only if it dfiffers.

File: libvirt/tests/src/cpu/guestpin.py [hunk 2]
```diff
@@ -109,24 +109,28 @@ def set_condition(vm_name, condn, reset=False, guestbt=None):
...
+                if not machine_cpuset_paths:
+                    logging.warning("cgroup cpuset might not recover properly "
+                                    "for guests after host smt changes, "
+                                    "restore it manually")
+                root_cpuset_cpus = os.path.join(root_cpuset_path, "cpuset.cpus")
```
Comment (chunfuwen) [inline libvirt/tests/src/cpu/guestpin.py]: it sound like the variable root_cpuset_cpus was never used

Comment (balamuruhans) [reply libvirt/tests/src/cpu/guestpin.py]: it is used at L128, L131

**Pull request merged**